### ISSUE TITLE
Add test for `heads` result field

### ISSUE THREAD

**Issue by ssiccha** [lbfm-rwth/GaussPar#77]
See #74. We can test this by comparing the output to the output of the Gauss package.

**Comment by emmakatherina** [comment]
I added all the tests and found that the output is generally false!

I tested with a 20x20 matrix of full rank and obtained the following information:
```
gap> Display(Concatenation(result_std.coeffs, result_std.relations) * A);
 1 . . . . . . . . . . . . . . . . . . .
 . 1 . . . . . . . . . . . . . . . . . .
 . . 1 . . . . . . . . . . . . . . . . .
 . . . 1 . . . . . . . . . . . . . . . .
 . . . . 1 . . . . . . . . . . . . . . .
 . . . . . 1 . . . . . . . . . . . . . .
 . . . . . . 1 . . . . . . . . . . . . .
 . . . . . . . 1 . . . . . . . . . . . .
 . . . . . . . . 1 . . . . . . . . . . .
 . . . . . . . . . 1 . . . . . . . . . .
 . . . . . . . . . . 1 . . . . . . . . .
 . . . . . . . . . . . 1 . . . . . . . .
 . . . . . . . . . . . . 1 . . . . . . .
 . . . . . . . . . . . . . 1 . . . . . .
 . . . . . . . . . . . . . . 1 . . . . .
 . . . . . . . . . . . . . . . 1 . . . .
 . . . . . . . . . . . . . . . . 1 . . .
 . . . . . . . . . . . . . . . . . 1 . .
 . . . . . . . . . . . . . . . . . . 1 .
 . . . . . . . . . . . . . . . . . . . 1
gap> Concatenation(result_std.coeffs, result_std.relations) = -Concatenation(result.coeffs, result.relations);
true
gap> result_std.heads;
[ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20 ]
gap> result.heads;
[ 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ]
gap> # heads is calculated using only the content of pivotrows and pivotscols:
gap> result.pivotrows;
[ 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1 ]
gap> result.pivotcols;
[ 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1 ]
```

So obviously the content of pivotrows and pivotcols is either false or my understanding of it is incorrect.

**Comment by ssiccha** [comment]
Since this seems to be non-trivial I'll move this to a later milestone.

**Comment by ssiccha** [comment]
The createHeads function is being fixed in #218.
